apps/foundation-demo/src/app/design/atoms/container/container.component.spec.ts

```diff
@@ -21,5 +21,21 @@ describe('ContainerComponent', () => {
 
   it('should create', () => {
     expect(component).toBeTruthy();
-  });
+  }); 
+  
+  fdescribe('setting the size of the container', () => {
+    it('should set a class on the host element depending on the size', () => {
+      component.size = 'small';
+      fixture.detectChanges();
+      expect(fixture.nativeElement.classList.contains('daff-container--small')).toEqual(true);
+
+      component.size = 'medium';
+      fixture.detectChanges();
+      expect(fixture.nativeElement.classList.contains('daff-container--medium')).toEqual(true);
+
+      component.size = 'large';
+      fixture.detectChanges();
+      expect(fixture.nativeElement.classList.contains('daff-container--large')).toEqual(true);
+    })
+  })
```

Comment:
These should be three different tests, like:
```
describe('setting the size of the container', () => {
  describe('when size is small', () => {
    it('should set "daff-container--small" on host element', () => {
    });
  });
});
```

each "it" should usually only test one thing. Also, the "fdescribe" needs to be changed to "describe"